src/firefly/js/templates/lightcurve/LcPhaseTable.js

```diff
@@ -23,6 +23,18 @@ const DEC_PHASE = 3;       // decimal digit
  */
 export function uploadPhaseTable(tbl, flux) {
     const {tbl_id, title} = tbl;
+    const data = tbl.tableData.data;
+
+    let badIndex=[];
+    let k=0;
+    for (let i=0; i<data.length; i++){
+        for(let j=0; j<data[i].length; j++){
+            if (data[i][j].startsWith('999999') ){
+                badIndex.push(k);
```

Comment:
I understand that by filtering out those numbers we fix the problem but why in the first place those number occured? Is it a side effect of the phase folding formula?

Reply:
Sorry I forgot to remove it.

Reply:
ok, great!